features/step_definitions/nfd.rb

```diff
@@ -84,7 +83,14 @@
   step %Q/all the pods in the project reach a successful state/
   step %Q/I store all worker nodes to the :worker_nodes clipboard/
   step %Q/I store the masters in the :master_nodes clipboard/
-  step %Q/<%= cb.master_nodes.count %> pods become ready with labels:/, table(%{
+
+  # for OCP >= 4.12, the number of master nodes with the label 'nfd-master' is
+  # 1
+  cb.expected_master_nodes_with_label = cb.master_nodes.count
+  if env.version_gt("4.11", user: user)
```

Comment:
Perhaps use `env.version_ge("4.12", user: user)` ?

Reply:
@liangxia sure, it's fixed now